### PR TITLE
Add systemd service for Arch

### PR DESCRIPTION
**Pull request by Kimiblock**

### CODE REVIEW

File: arch/conduwuit.service [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+RemoveIPC=yes
+RestrictAddressFamilies=AF_INET AF_INET6 AF_UNIX
+RestrictNamespaces=yes
+RestrictRealtime=yes
+RestrictSUIDSGID=yes
+SystemCallArchitectures=native
+SystemCallFilter=@system-service @resources
+SystemCallFilter=~@clock @debug @module @mount @reboot @swap @cpu-emulation @obsolete @timer @chown @setuid @privileged @keyring @ipc
+SystemCallErrorNumber=EPERM
+StateDirectory=matrix-conduit
```
Comment (girlbossceo) [inline arch/conduwuit.service:41]:
```suggestion
StateDirectory=conduwuit
```

File: arch/conduwuit.service [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+RuntimeDirectory=conduwuit
+RuntimeDirectoryMode=0750
+
+Environment="CONDUIT_CONFIG=/etc/conduwuit/conduwuit.toml"
```
Comment (girlbossceo) [inline arch/conduwuit.service:46]:
```suggestion
Environment="CONDUWUIT_CONFIG=/etc/conduwuit/conduwuit.toml"
```

**Review by girlbossceo — CHANGES_REQUESTED** [review]
Since I assume this will be hashed in the PKGBUILD, would like to just fix 2 things so I don't have to notify downstream when I change this.

File: arch/conduwuit.service (hunk 1 above)
Comment (Kimiblock) [reply arch/conduwuit.service:41]: I have to make sure it doesn't brick existing installation in the PKGBUILD before this can be changed

Comment (girlbossceo) [reply arch/conduwuit.service:41]: If `StateDirectory=` can be set multiple times, I'm okay with having both there. If it helps.

Comment (Kimiblock) [reply arch/conduwuit.service]: I'm testing with new .install script, which will automatically move directories into the right spot

**Comment by Kimiblock** [comment]
I'm waiting someone running conduwuit on Arch to test the new version of PKGBUILD.

This systemd service will be deleted from PKGBUILD once conduwuit issues a release.

**Pull request merged**